x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/rule_status.tsx

```diff
@@ -28,6 +28,7 @@ interface BaseProps {
   fieldName: string;
   isDraggable: boolean;
   value: string | number | undefined | null;
+  timelineId?: string;
```

Comment:
Random thought, not necessarily related to this line of code. Do we know all the timelineIds? What I'm trying to say is, can we make this type definition more narrow. Instead of `string` it could be `type TimelineId = 'oneId' | 'anotherOne'`. Wdyt?

Reply:
Yes, I agree, it should just be the enum here https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/common/types/timeline/index.ts#L314 instead of a string, but I started to change it once upon time and just stopped, as it was just a random string in a whole lot of test files etc.